app/src/main/java/com/example/whattowatch/fragments/MovieListFragment.java

```diff
@@ -52,5 +68,37 @@ public void onClick(View view) {
             }
         });
 
+        movieLists = new ArrayList<>();
+
+        // create adapter for RecyclerView
+        movieAdapter = new MovieListAdapter(getContext(), movieLists);
+
+        // set the adapter as the recycler view adapter
+        rvMovieLists.setAdapter(movieAdapter);
+
+        // Set a layout Manager on the recycler view
+        rvMovieLists.setLayoutManager(new LinearLayoutManager(getContext()));
+
+    }
+
+    protected void queryLists() {
+        // Specify which class to query
+        ParseQuery<MovieList> query = ParseQuery.getQuery(MovieList.class);
+        query.include(MovieList.KEY_USER);
+        query.addDescendingOrder(MovieList.KEY_CREATED_AT);
+        query.findInBackground(new FindCallback<MovieList>() {
+            @Override
+            public void done(List<MovieList> movieLists, ParseException e) {
```

Comment:
you can override functions for a specific instance of a class? 😮

Reply:
yes, when defining a FindCallback, you have to override and define the done method, as that is typically instance specific